pandas/types/common.py

```diff
@@ -59,6 +62,17 @@ def is_sparse(array):
     return isinstance(array, (ABCSparseArray, ABCSparseSeries))
 
 
+def is_scipy_sparse(array):
+    """ return if we are a scipy.sparse.spmatrix """
```

Comment:
Found it here: https://wiki.python.org/moin/PythonSpeed/PerformanceTips#Import_Statement_Overhead Was worried about the same thing, but have tested in Py2 and Py3 that the global is indeed set correctly.